config.example.ini

```diff
@@ -40,6 +40,12 @@ enable=false
 ;;; Verschiedene Benachrichtigungsservices (So ziemlich alles was man sich vorstellen kann). Mehrere URIs durch Kommata trennen -> https://github.com/caronc/apprise
 service_uris=discord://webhook_id/webhook_token,matrix://hostname
 
+[ADMIN]
+;;; Benachrichtigungsweg für Fehlermeldungen an den Admin:
+;;; 'email', 'telegram', [...] oder 'none', wenn keine separaten Fehlermeldungen gesendet werden sollen
+method=none
```

Comment:
@sibalzer my proposal to avoid a huge config.ini or cluttering the directory with a second config file

Reply:
i still have to think about it. it would make sense to have three variables where you can specify either a mail address telegram id or apprise uri where a notification can be sent to. I would write this then rather only in the documentation to not create confusion.

e.g.

```ini
[ADMIN]
notification_telegram_id=
notification_email_adress=
notification_aprrise_uri=
```

Reply:
btw i dont think we need a beta or dev branch anymore. with v3 this was still necessary because there were massive feature addings and i made big changes with settings.py in parallel. i just couldn't handle all the manual testing. but that changed and we have a ci pipeline with linting, unittesting and integration tests. we can be more agile now.

Reply:
Your proposal looks great, I like it. I was also thinking about it, but apparently not far enough.

So do you want to merge new feature branches directly onto main now? I wasn't sure, but if you say it should be fine, we can do that of course.

Reply:
We should approach it that way. But I don't have time at the moment (exams), so no pressure.